### DISCUSSION TITLE
Posibility to adjust vertical spacing

### DISCUSSION THREAD

**Discussion by SorinGFS** [Kong/insomnia#3937] [Ideas]
It would be nice if we had the possibility to adjust the vertical spacing between the rows in interface and text editor. I tried to lower the font size in order to have more records in view, but due to the fixed spaging it wasn't too helpful.

**Comment by dimitropoulos** [comment]
I'm not sure where you're talking about (there are a few different places that (to me) fit the description you gave).  Could you please provide a screenshot?

**Reply by SorinGFS** [reply]
Sure.
![Untitled](https://user-images.githubusercontent.com/25417015/131526493-7cce8133-f8dd-42a4-827c-93fbe1a6e1e2.png)

**Reply by dimitropoulos** [reply]
ahhh, perfect.  Thank you.  @erickrico any ideas.  I suppose there's always the option of adding a setting for adjusting this.  I can totally see how someone may think it to be too much.  Actually, now that I see this mockup _I_ kinda think it's too much, heh.

**Reply by SorinGFS** [reply]
I really appreciate your efforts, but I think this is a must have feature, The whole height takes only 19 rows, I think this is a waist of space, see this: 
![3](https://user-images.githubusercontent.com/25417015/131535100-c090a050-d975-43ba-bbb7-d19ab320e24e.png)

**Comment by dimitropoulos** [comment]
Does zooming out make any difference (`Ctrl` + `-` and `Ctrl` + `=`)?

**Reply by SorinGFS** [reply]
The zoom changes the size of the font in editor and preview panes, I don't want that.

**Reply by dimitropoulos** [reply]
thanks for confirming!

**Comment by dimitropoulos** [comment]
I was looking for how vscode does this and all I found was https://github.com/microsoft/vscode/issues/59873.  If anyone has an example of another app that does this it's always good to compare.  Of course, it's a straightforward _enough_ feature, but there are always questions to answer like "should it be configurable with a number or should it have certain settings like `compact` and `relaxed` and whatnot".

Personally, I'm a bigger fan of the way that gmail does this:
![mail google com_mail_u_0_](https://user-images.githubusercontent.com/15232461/131529438-12a77189-f9a6-4196-8d43-479762f43628.png)

**Reply by SorinGFS** [reply]
The google approach LGTM, but I'm not against the version with configurable with a number like in css line height.

**Reply by erickrico** [reply]
sounds good, i can go either way, or even both, like a custom option at the bottom
